src/handlers/comments/getComments.js

```diff
@@ -9,12 +9,7 @@ async function getComments(request, h) {
         .then((data) => {
```

Comment:
> Wouldn't we still need to loop even if we chained another `.then()`?

Yep, I meant to have another `.then()` with a `.map()` inside before this. That would allow us to get rid of the extra function. Something like this:

```js
.then(data => data.map(item => /* … */))
.then(data => {
    // switch…
})
```

Reply:
Updated.